website/static/js/tests/osfHelpers.test.js

```diff
@@ -0,0 +1,190 @@
+/*global describe, it, expect, example, before, after, beforeEach, afterEach, mocha, sinon*/
+'use strict';
+var assert = require('chai').assert;
+var $ = require('jquery');
+var moment = require('moment');
+var Raven = require('raven-js');
+
+var $osf = require('../osfHelpers');
+
+// Add sinon asserts to chai.assert, so we can do assert.calledWith instead of sinon.assert.calledWith
+sinon.assert.expose(assert, {prefix: ''});
+
+describe('osfHelpers', () => {
+    describe('growl', () => {
+        it('calls $.growl with correct arguments', () => {
+            var spy = new sinon.spy($, 'growl');
+            $osf.growl('The one', 'the only', 'danger');
+            assert.calledOnce(spy);
+
+            assert.calledWith(spy,
+                {title: '<strong>The one<strong><br />', message: 'the only'});
+            spy.restore();
+        });
+    });
+
+    describe('handleJSONError', () => {
+        var response = {
+            responseJSON: {message_short: 'Oh no!', message_long: 'Something went wrong'}
+        };
+        it('uses the response body if available', () => {
+            var spy = new sinon.spy($osf, 'growl');
+            $osf.handleJSONError(response);
+            assert.called(spy);
```

Comment:
Does this actually test that it's using the response body if available? Or just that the growlbox was called when you handle a json error?